mobile/lib/features/welcome/welcome_screen.dart

```diff
@@ -74,9 +74,9 @@ class _WelcomeScreenState extends State<WelcomeScreen> {
                   onPressed: () {
                     if (_formKey.currentState != null && _formKey.currentState!.validate()) {
                       _formKey.currentState?.save();
+                      api.registerBeta(email: _email);
```

Comment:
🙃 It appears I missed this in a previous review: 
We should avoid `api` calls in the user interface!

It would be great if you could clean this up and wrap it in a service.
I think the service call should be wrapped in a `try catch` here so we are exiting gracefully.

Reply:
I'll create a refactor ticket out of this, right now it's not the priority